src/Helpers/Rules.elm

```diff
@@ -324,161 +343,220 @@ sups txt =
-                , binaryExample "(1) F(a↔b) [ ]" "(2) F T (a∧¬b) [1]" "(3) F T (a∨¬b) [1]"
-                , linearExample "" "(1) (1) F ∀x∀y P(x,y) [ ] [ ]" "(2) F P(q,z) {x→q, y→z} [1]"
-                , linearExample "" "(1) T ∀x∀y P(x,y) [ ]" "(2) T P(g(k,z), q) {x→g(k,z) ,y→q} [1]"
-                ]
+cutItem config =
+    ruleItem
+        "Cut"
+        (List.map renderBinary cutFormulas)
+        (binaryExample "" "T a [ ]" "F a [ ]")
+        config
+
+
+modusPonensItem config =
+    ruleItem
+        "MP"
+        (List.map renderUnary mpFormulas)
+        (linearExample "(1) T(a→b) [ ]" "(2) T a [ ]" "(3) T b [1,2]")
+        config
+
+
+modusTolensItem config =
+    ruleItem
+        "MT"
+        (List.map renderUnary mtFormulas)
+        (linearExample "(1) T(a→b) [ ]" "(2) F b [ ]" "(3) F a [1,2]")
+        config
+
+
+hsItem config =
+    ruleItem
+        "HS"
+        (List.map renderUnary hsFormulas)
+        (linearExample "(1) T(a→b) [ ]" "(2) T(b→c) [ ]" "(3) T(a→c) [1,2]")
+        config
+
+
+dsItem config =
+    ruleItem
+        "DS"
+        (List.map renderUnary dsFormulas)
+        (linearExample "(1) T(a∨b) [ ]" "(2) F a [ ]" "(3) T b [1,2]")
+        config
+
+
+ncsItem config =
+    ruleItem
+        "NCS"
+        (List.map renderUnary ncsFormulas)
+        (linearExample "(1) F(a∧b) [ ]" "(2) T a [ ]" "(3) F b [1,2]")
+        config
+
+
+esttItem config =
+    ruleItem
+        "ESTT"
+        (List.map renderUnary esttFormulas)
+        (linearExample "(1) T(a↔b) [ ]" "(2) T b [ ]" "(3) T a [1,2]")
+        config
+
+
+estfItem config =
+    ruleItem
+        "ESTF"
+        (List.map renderUnary estfFormulas)
+        (linearExample "(1) T(a↔b) [ ]" "(2) F a [ ]" "(3) F b [1,2]")
+        config
+
+
+esftItem config =
+    ruleItem
+        "ESFT"
+        (List.map renderUnary esftFormulas)
+        (linearExample "(1) F(a↔b) [ ]" "(2) T a [ ]" "(3) F b [1,2]")
+        config
+
+
+esffItem config =
+    ruleItem
+        "ESFF"
+        (List.map renderUnary esffFormulas)
+        (linearExample "(1) F(a↔b) [ ]" "(2) F b [ ]" "(3) T a [1,2]")
+        config
+
+
+ecdtItem config =
+    ruleItem
+        "ECDT"
+        (List.map renderBinary ecdtFormulas)
+        (binaryExample "(1) T(a↔b) [ ]" "(2) T(a∧b) [1]" "(3) F (a∨b) [1]")
+        config
+
+
+ecdfItem config =
+    ruleItem
+        "ECDF"
+        (List.map renderBinary ecdfFormulas)
+        (binaryExample "(1) F(a↔b) [ ]" "(2) F T (a∧¬b) [1]" "(3) F T (a∨¬b) [1]")
+        config
+
+
+rulesTable config =
+    div [ class "rules-container" ] <|
+        List.filterMap (\a -> a config)
+            [ alphaItem
+            , betaItem
+            , gammaItem
+            , deltaItem
+            , gammaStarItem
+            , deltaStarItem
+            , reflexivityItem
+            , leibnitzItem
+            , modusPonensItem
+            , modusTolensItem
+            , hsItem
+            , dsItem
+            , ncsItem
```

Comment:
Cut is missing. Also, since DS and NCS are similar to MP, MT, I’d suggest ordering the rules in this section as follows:
- DS
- NCS
- Cut
- HS